notebooks/week5/01.preprocess.py

```
@@ -0,0 +1,94 @@
+
+# Verify affected rows count for train and test
+affected_rows_train = new_data_train.count()
+affected_rows_test = new_data_test.count()
+
+if affected_rows_train > 0 or affected_rows_test > 0 :
+    refreshed = 1
+else:
+    # simulate refresh
+    refreshed = 1
+
+dbutils.jobs.taskValues.set(key="refreshed", value=refreshed)
```

Ruff

Undefined name `dbutils`

(F821)